gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberApi.java

```diff
@@ -66,9 +65,26 @@
  */
 @javax.annotation.Generated("by GAPIC")
 public class SubscriberApi implements AutoCloseable {
+  // ========
```

Comment:
Less likely by the 3-way merge process.

They are really just a holdover from the coding style of the first prototype; I'll see what the team thinks and possibly remove them in the next version (which will have other style fixes).